recipes/fast-cdr/all/test_package/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    skip_build = False
+    skip_testing = False
+
+    def build(self):
+        if self.settings.os == "Windows":
+            if (str(self.settings.compiler.runtime).startswith("MT") and self.options['fast-cdr'].shared):
```

Comment:
This means the library is not produced correctly... raise in the recipe so consumers do not have something unusable